src/Emogrifier.php

```diff
@@ -1003,7 +1003,7 @@ private function copyUninlinableCssToStyleNode(array $cssRules, $cssImportRules)
         // avoid including unneeded class dependency if there are no rules
         if ($cssRulesRelevantForDocument !== []) {
             // support use without autoload
-            if (!\class_exists('Pelago\\Emogrifier\\Utilities\\CssConcatenator')) {
+            if (!\class_exists(CssConcatenator::class)) {
```

Comment:
This won't work - if autoloading is not used (which is the reason for its existence in the first place), it will cause an error that `CssConcatenator` is not defined.

Reply:
Yes, this will work. The `::class` notation is only a shortcut for the name (as imported via the `use` statement above or the current namespace). This also works even if the class cannot be loaded or does not exist because this is on the naming level only.

Reply:
OK.  You're right.  (For peace of mind, I tested and confirmed, at least for PHP 7.3.)